### PR TITLE
Portfolio webiste 2

### PR DESCRIPTION
**Pull request by den-cloud**
In this pull request ,I added the last part of the mobile version which is the contact form.

I used html and background to the necessary information required and used using css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -189,7 +196,21 @@ <h3>Skills</h3>
...
-    </main>
+    </section>
+
+    <section class="contact-section">
+        <h1 id="contact-card-title">Contact</h1>
+
+        <p>If you have an application you are interested in developing, a feature that you need
+            built or a project that needs coding. I’d love to help with it </p>
+
+        <form action="#" method="POST" class="contact-form"></form>
```
Comment (karvel-code) [inline index.html]: - [ ] Please ensure that you add the formspree link as the value to the action attribute. The link should be similar to: `https://formspree.io/f/{form_id}`.
- [ ]  Ensure that your input tags are wrapped inside the form tags. The `</form>` closing tag should be after the button code(after line 212 and before line 213).

File: index.html [hunk 2]
```diff
@@ -189,7 +196,21 @@ <h3>Skills</h3>
...
+        <input type="text" name="_name" placeholder="Name" class="form-field" required>
+        <input type="email" name="_replyto" placeholder="Email" class="form-field" required>
+        <textarea name="_message" placeholder="Write your message here" maxlength="500"
+            class="form-field contact-message"></textarea>
+        <button type="submit" id="contact-btn">Get In Touch</button>
```
Comment (karvel-code) [inline index.html]: - [ ] Your form is supposed to submit valid data successfully. However, this is not the case with your form. Please work on that issue.

File: index.html [hunk 3]
```diff
@@ -189,7 +196,21 @@ <h3>Skills</h3>
...
+    </section>
+
+    <section class="contact-section">
+        <h1 id="contact-card-title">Contact</h1>
+
+        <p>If you have an application you are interested in developing, a feature that you need
+            built or a project that needs coding. I’d love to help with it </p>
+
+        <form action="#" method="POST" class="contact-form"></form>
+        <input type="text" name="_name" placeholder="Name" class="form-field" required>
```
Comment (karvel-code) [inline index.html]: - [ ] Please ensure to add the attribute `maxlength = "30" ` for the name input tag. This is so as to add and improve on the validations for your form.

File: index.html [hunk 4]
```diff
@@ -189,7 +196,21 @@ <h3>Skills</h3>
...
+        <input type="email" name="_replyto" placeholder="Email" class="form-field" required>
+        <textarea name="_message" placeholder="Write your message here" maxlength="500"
+            class="form-field contact-message"></textarea>
```
Comment (karvel-code) [inline index.html]: - [ ] Please ensure to add the `required` attribute for the message input. This is in order to activate its validation on submit.

File: index.html (hunk 2 above)
Comment (karvel-code) [reply index.html]: - [ ] Please ensure you follow the design specified on Figma for the button.

**Review by karvel-code — CHANGES_REQUESTED** [review]
Hi @den-cloud , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Your linter checks pass successfully.

## Required Changes ♻️
_Check the comments under the review._

- Please ensure you follow the design provided in the Figma design for the contact form section:

- [ ] Check on the margins between the different features of the page.
- [ ] Also, check on the font required according to the design especially for the placeholder texts.
- [ ] Ensure to use the right colours required. For instance, the colour for the `Contact me` title is not black as done according to your design.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] Please enhance the responsiveness of your site. The content is not positioned in the right manner for some mobile sizes.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by dasileker — APPROVED** [review]
### **Hi @den-cloud,** 

### **Project Approved 🎉✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**